bin/tic-tac-toe.rb

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env ruby
+require '../lib/board.rb'
+require '../lib/player.rb'
+
+puts "\n"
```

Comment:
To make tic-tac-toe.rb cleaner I suggest transferring the code for generating the UI into another file in its own method which you will just invoke here